src/renderer/atlas/AtlasEngine.r.cpp

```diff
@@ -672,11 +661,19 @@ AtlasEngine::CachedGlyphLayout AtlasEngine::_getCachedGlyphLayout(const wchar_t*
     // where every single "=" might be blatantly misaligned vertically (same for any box drawings).
     WI_SetFlagIf(options, D2D1_DRAW_TEXT_OPTIONS_NO_SNAP, scalingRequired);
 
+    // ClearType basically has a 3x higher horizontal resolution. To make our glyphs render the same everywhere,
+    // it's probably for the best to ensure we initially rasterize them on a whole pixel boundary.
+    // (https://en.wikipedia.org/wiki/ClearType#How_ClearType_works)
+    offset.x = roundf(offset.x * _r.pixelPerDIP) * _r.dipPerPixel;
+    // As explained below, we use D2D1_DRAW_TEXT_OPTIONS_NO_SNAP to prevent a weird issue with baseline snapping.
+    // But we do want it technically, so this re-implements baseline snapping... I think?

```

Comment:
reviewer note: comment is not new, don't think too hard about the `... I think?"